src/huggingface_hub/README.md

```diff
@@ -81,6 +40,29 @@ Parameters:
 - a `cache_dir` which you can specify if you want to control where on disk the
   files are cached
 
+### `hf_hub_url`
+
+Internally, the library uses `hf_hub_url()` to return the URL to download the actual files:
```

Comment:
```suggestion
Internally, the library uses [`hf_hub_url`] to return the URL to download the actual files:
```

I think this links to the package reference page

Reply:
if i'm not mistaken this file is not rendered in the doc (it's just displayed in GitHub) so i'll reject this proposal